VTUL/dlp-access
# Fix bug relating to tags not having predefined labels

**Fix bug relating to tags not having predefined labels.**
* * *

**JIRA Ticket**: (None, sorry) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Fixes bug relating to tags not having predefined labels. Tags are not included in "search facets" because they are defined IN the metadata and can't be predefined with an associated label. So the label for ANY tag should be "Tag". Actual facets should continue to render their labels as defined.
This PR also includes an update to `package.json` to address a bug caused by competing versions of `react-refresh`

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.
* Adds a function to SearchResults.js to handle separate case for tag label in addition to facet case
* `package.json` will force `react-refresh` node package to 0.14.0 in to eliminate competing versions that are breaking my local installation and really getting on my nerves.

# How should this be tested?
* In the generated preview site (I'll add when it's built) go to the search page '/search'
* Click on any item in the search results marked "Item" not "Collection"
* Click on any tag under the "Tags" metadata field.Not every item has a tag/tags. Might have to try a few different items?
* Check that the page renders correctly now.
* Check that the search results also contain the tag that was clicked

# Additional Notes:
* branch: `tag_label_fix`

# Interested parties
@otokama 
@goynejennifer 

(:star:) Required fields
